database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala

```diff
@@ -14,5 +15,17 @@ class MetadataDataAccessComponent(val driver: JdbcProfile) extends DataAccessCom
       customLabelEntries.schema ++
       metadataEntries.schema ++
       summaryStatusEntries.schema ++
-      workflowMetadataSummaryEntries.schema
+      workflowMetadataSummaryEntries.schema ++
+      summaryQueueEntries.schema
+
+  // Looks like here is the most appropriate place for this val since it doesn't fit neither in
+  // SummaryQueueEntryComponent nor in MetadataEntryComponent
```

Comment:
This looks like the functions in `SummaryQueueSlickDatabase` to me

Reply:
That was my initial thought, but i wasn't able to find any single example where we currently mix two different tables in one of those `<TableName>SlickDatabase` classes. Also having it as a function, will cost us benefits of having it as a `Compiled` statement in a val.